src/miner.h

```diff
@@ -146,6 +146,7 @@ class BlockAssembler
     int64_t nLockTimeCutoff;
     const CChainParams& chainparams;
     const CTxMemPool& m_mempool;
+    CChainState& m_chainstate;
```

Comment:
nit: Can this be const?

Reply:
Unfortunately not... We pass `m_chainstate` to `TestBlockValidity` in `BlockAssembler::CreateNewBlock`, and `TestBlockValidity` requires its `CChainState` param to be non-const